ExtendedDescription.md

```diff
@@ -1,121 +1,154 @@
-## About nelson.social
+# nelson.social
+
+## Server Rules
+
+nelson.social is an online space dedicated to the community of Nelson, BC. Please carefully read our Terms of Use as set out on this page before using this online space. These terms govern your use of the nelson.social online space and by using the nelson.social online space, you agree to be bound by these terms.
```

Comment:
I might be wrong, but I don't think Mastodon gives you a way to customise this preamble to the server rules (unless we fork it 😈)

So if this is important, we might want to put it somewhere in the `About` section which we do have control over.

Reply:
Yeah, that's how my lawyer structured it. I can either patch this, or we leave it out and we use this documents order for #24?

Reply:
Why don't we try and get #24 working on the cloudflare end, then we can iterate on the content in there?

Reply:
Although I think ideally we'd put this particular paragraph in front of people right as they sign up. I think the easiest way to do that would be to make it a server rule?

Reply:
I'll use a Cloudflare Worker to inject it as HTML.